modules/aws/aws.bash.rc

```diff
@@ -0,0 +1,2 @@
+source ~/.bashrc
+PS1=${PS1}"(aws mode)$ "
```

Comment:
Consider emitting information about the session so they know the role they assumed:

https://github.com/cloudposse/geodesic/blob/master/rootfs/etc/profile.d/aws-vault.sh#L5

Reply:
or simply echo `AWS_PROFILE` here